process/living-cr-rec.html

```diff
@@ -0,0 +1,121 @@
+	padding-left: 1em;}
+dl.time,span.time {
+	border: 2px solid #C99;
+	background: #EE9; }
+dl.time {
+	padding: 1em }
+dl.time > dt:before {
+	content: "Time: "}
+dl.tip {
+	padding: 1em }
+dl.tip,span.tip {
+	border: 2px solid #9C9;
+	background: #BEB; }
+dl.tip > dt:before {
+	content: "Tips: "}
+</style>
+  </head>
+  <body>
+    <div id="header"> <span class="logo"><a href="https://www.w3.org/"><img src="/Icons/WWW/w3c_home_nb"
+            alt="W3C" height="48" width="72"/></a></span>
+      <div class="breadcrumb"> <a href="https://www.w3.org/Guide/">The Art of
+          Consensus</a> →
+        <h1>Candidate Recommendation Snapshots and Recommendations</h1>
+      </div>
+      <p class="baseline">This <strong>Guidebook</strong> is the collected
+        wisdom of the W3C Group Chairs and other collaborators.</p>
+    </div>
+    <!-- <div class="toc">
+      <h4>Also On This Page → </h4>
+      <ul>
+        <li><a href="#prep">Preparatory Work</a> •</li>
+        <li><a href="#wg-startup">Starting the Group</a> •</li>
+        <li><a href="#fpwd">First Public Working Draft</a> •</li>
+        <li><a href="#test">Test suites</a> •</li>
+        <li><a href="#getting-to-lc">Getting to Candidate Recommendation</a> •</li>
+        <li><a href="#processing-lc">Processing Candidate Recommendation</a> •</li>
+        <li><a href="#transition-cr">Transition to CR</a> •</li>
+        <li><a href="#leaving-cr">Leaving CR</a> •</li>
+        <li><a href="#pr">Proposed Recommendation</a> •</li>
+        <li><a href="#rec">Recommendation</a> •</li>
+      </ul>
+    </div> -->
+    <div class="main">
+      <h2 id="intro">Introduction</h2>
+
+      <p>
+        This document is intended to help Working Groups in deciding whether to
+        maintain a document using revised Candidate Recommendations or revised Recommendations.
+      </p>
+      <h2 id="process">W3C Process</h3>
+      <p>The W3C Process offers two ways to maintain a document on the Recommendation track:</p>
+      <dl>
+        <dt><a href="https://www.w3.org/Consortium/Process/#revising-cr">Revising a Candidate Recommendation</a></dt>
+        <dd>
+          Candidate Recommendation Snapshots get similar patent commitments to W3C Recommendations but do not get
+          endorsed by W3C as standards for the Web. The document can be revised rapidly by publishing Candidate
+          Recommendation Drafts on a daily basis.
+        </dd>
+        <dt><a href="https://www.w3.org/Consortium/Process/#revising-rec">Recommendations</a></dt>
+        <dd>
+          Recommendations are endorsed by W3C as standards for the Web. The document can be revised rapidly by publishing candidate
+          corrections/additions on a daily basis.
+        </dd>
+      </dl>
+
+      <h2 id="cr">When should a Group keep a document as a Candidate Recommendation?</h2>
+
+        <p>
+          A document remains in Candidate Recommendation if one or more of the conditions apply:
```

Comment:
"When should" suggests that in some cases the preferred choice is to stay in CR. But the text just describes the scenarios in which a document cannot proceed to a higher maturity level.

I would like to see:

1. a clear steer towards getting to Rec if possible
2. a steer towards reverting to Note if it looks like there's no realistic chance of ever moving forward to Rec.

I can't see how it helps anyone to have specifications remaining at CR semi-permanently because the criteria for advancement are never going to be met.

_Especially_ given that if there is even a subset of features that can make it to Rec, with the remainder stuck in CR, there's now a mechanism for managing that.

Reply:
> "When should" suggests that in some cases the preferred choice is to stay in CR. But the text just describes the scenarios in which a document cannot proceed to a higher maturity level.
> I would like to see:
> 
> 1. a clear steer towards getting to Rec if possible

I added a section on "When should a Group move to Recommendation" ?

> 2. a steer towards reverting to Note if it looks like there's no realistic chance of ever moving forward to Rec.
> 
> I can't see how it helps anyone to have specifications remaining at CR semi-permanently because the criteria for advancement are never going to be met.

Trickier, since moving to Note track means moving out of the Patent Policy. So, unless it becomes impossible to resolve technical issues, I don't think we would want to advise a Group to move a document out of the REC track.
 
> _Especially_ given that if there is even a subset of features that can make it to Rec, with the remainder stuck in CR, there's now a mechanism for managing that.

Reply:
> I don't think we would want to advise a Group to move a document out of the REC track.

Sorry, my mistake. I was remembering the older Process that (I think) instructed dropping back to Note; it now says [§6.3.12.1](https://www.w3.org/2021/Process-20211102/#abandon-draft) to move it to Discontinued Draft. So my previous comment but s/Note/Discontinued Draft.

Reply:
> I added a section on "When should a Group move to Recommendation" ?

Thanks, works for me!